src/ConnectedMode/ConnectedMode.csproj

```diff
@@ -1,13 +1,16 @@
 <Project Sdk="Microsoft.NET.Sdk">
   
   <Import Project="..\SonarLint.Product.props" />
+  <Import Project="..\SonarLint.VsPkgRefs.props" />
 
   <PropertyGroup>
     <ProjectGuid>0BE551DB-3C46-42A5-BB38-DA80E83F8ABD</ProjectGuid>
     <AssemblyName>SonarLint.VisualStudio.ConnectedMode</AssemblyName>
     <RootNamespace>SonarLint.VisualStudio.ConnectedMode</RootNamespace>
 
     <RequiresLibGit>true</RequiresLibGit>
+
+    <LangVersion>8</LangVersion>
```

Comment:
Some newer language features are implemented solely by the compiler, but some also rely on a particular target framework. For product code, we don't want to be in the situation where we accidentally use a language feature that relies on a .NET runtime version that's newer than the one used by VS2019.3.

To be on the safe side, we only set the language version for test projects.

Which C#8 languages features are you using?